courses/programming-in-haskell/pih-exercises/test/Chapter06Spec.hs

```diff
@@ -1,5 +1,3 @@
--- | Specification for the exercises of Chapter 6.
```

Comment:
clean code; it does not add anything; if it does change the module or directory names